packages/lexical-playground/src/nodes/ExcalidrawNode/index.tsx

```diff
@@ -48,10 +48,7 @@ function $convertExcalidrawElement(
     !widthStr || widthStr === 'inherit' ? 'inherit' : parseInt(widthStr, 10);
 
   if (excalidrawData) {
-    const node = $createExcalidrawNode();
-    node.__data = excalidrawData;
-    node.__height = height;
-    node.__width = width;
+    const node = new ExcalidrawNode(excalidrawData, width, height);
```

Comment:
The original was more correct, it's always preferable to use a constructor function so that `$applyNodeReplacement` can be used

```suggestion
    const node = $createExcalidrawNode(excalidrawData, width, height);
```